core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -69,13 +76,13 @@
 public class ArquillianCucumber extends Arquillian {
     private static final Logger LOGGER = Logger.getLogger(ArquillianCucumber.class.getName());
 
-    private static final String RUN_CUCUMBER_MTD = "____Cucumber_Runner_Not_A_Test";
```

Comment:
I'm not very motivated for this change, if you make this name even more complicated I'd take it. Idea is we would like to completely hide this method but it is here to avoid to make this class very complicated so it must not follow java rules.